docs/content/quickstart.md

```diff
@@ -38,6 +30,174 @@ helm installation to perform all steps it needs is to edit the
 Assuming `tiller` is deployed in your `kube-system` namespace, this will
 give it very open permissions.
 
-### Installation Examples
+## Installing Maesh
+
+```bash tab="Command"
+helm repo add maesh https://containous.github.io/maesh/charts
+helm repo update
+helm install --name=maesh --namespace=maesh maesh/maesh
+```
+
+## Using Maesh
+
+Let's deploy a server application and a client application under the `maesh-test` namespace.
```

Comment:
Maybe we should make it more obvious that this is an example and not the way they have to use Maesh 😅

```suggestion
As an example, let's deploy a server application and a client application under the `maesh-test` namespace.
```